.gitignore

```diff
@@ -10,3 +10,4 @@ vendor
 /internal/rust/parser/pkg
 /internal/rust/site/node_modules
 /internal/rust/.cache
+.DS_Store
```

Comment:
Can you remove this change and instead add this a global gitignore on your laptop? See https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Yeah,  no problem. I can update that.